llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.h

```diff
@@ -814,6 +814,15 @@ bool shouldEmitConstantsToTextSection(const Triple &TT);
 /// to integer.
 int getIntegerAttribute(const Function &F, StringRef Name, int Default);
 
+/// \returns Unsigned Integer value requested using \p F's \p Name attribute.
+///
+/// \returns \p Default if attribute is not present.
+///
+/// \returns \p Default and emits error if requested value cannot be converted
+/// to integer.
+unsigned getUnsignedIntegerAttribute(const Function &F, StringRef Name,
```

Comment:
The above getIntegerAttribute is actually dead, this shouldn't follow along. The equivalent functionality was moved to getFnAttributeAsParsedInteger

Reply:
Removed `getUnsignedIntegerAttribute()`.